contracts/MockContract.sol

```diff
@@ -1,4 +1,6 @@
-pragma solidity ^0.6.0;
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Shouldn't we use the real licence identifier here. I thought `UNLICENSED` is for non-open-source (see https://forum.openzeppelin.com/t/solidity-0-6-8-introduces-spdx-license-identifiers/2859)

Reply:
This would require us to chose a Licence right? Last time I spoke with legal I heard it was better to not use a licence (due to some issues with licence of dependencies.

Reply:
Reached out again, but technically this is unrelated to this PR.